app/components/viewer/preload.js

```diff
@@ -1,4 +1,5 @@
 const { ipcRenderer } = require('electron')
+const { EventEmitter2 } = require('eventemitter2')
```

Comment:
Why not use `require('events').EventEmitter`?

Reply:
I didn't realize that was already coming in as a dependency, used that